custom/icds_reports/reports/incentive.py

```diff
@@ -65,12 +70,16 @@ def _format_infrastructure_data(data):
                     AWC_NOT_LAUNCHED
                 ])
             else:
-                if self.month.year < 2019 or (self.month.year == 2019 and self.month.month < 3):
-                    func = int
+                if self.beta:
+                    visit_denominator = row_data['visit_denominator']
                 else:
-                    func = round
-                home_visit_percent = row['valid_visits'] / func(row['expected_visits']) if \
-                    func(row['expected_visits']) else 1
+                    if self.month.year < 2019 or (self.month.year == 2019 and self.month.month < 3):
```

Comment:
a comment here would be helpful

Reply:
Same comment as above. we are removing this hopefully within the week but can add a comment in the meantime if necessary